app/brave_generated_resources.grd

```diff
@@ -645,6 +645,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_ADS_NOTIFICATION_AD_ACCESSIBLE_NAME" desc="Text to be spoken when a notification ad is shown">
         Brave Notification Ad
       </message>
+      <message name="IDS_BRAVE_ADS_NOTIFICATION_EXTERNAL_WALLET_CONNECTED" desc="The notification to show when an external wallet is connected">
```

Comment:
Will this text fit in both system and custom push notifications?

Reply:
It doesn't fit in the custom notification. I was thinking that what I can do for custom notifications is pull the "yay" emoji and first sentence into the title and keep everything else as the body.

![custom](https://user-images.githubusercontent.com/5033691/229630676-c4f3fc3c-e93e-42b0-995a-9197f057b272.png)

I'm not sure why there's so much empty space above the title, though (unless that's just what the custom notification UI looks like on Windows?)

Reply:
Empty space is expected for custom notifications, per Terry.